src/pages/en/guides/markdown-content.md

```diff
@@ -277,40 +324,21 @@ const content = await fetch('https://raw.githubusercontent.com/withastro/docs/ma
 
 ⚠️ Use of the `Markdown` component to render remote Markdown can open you up to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack. If you are rendering untrusted content, be sure to _sanitize your content **before** rendering it_.
 
+## Markdown Configuration
 
-## Markdown Parsers
-
-Astro comes with Markdown support powered by [remark](https://remark.js.org/). 
-
-The `@astrojs/markdown-remark` package is included by default with the following plugins pre-enabled:
-
-- [GitHub-flavored Markdown](https://github.com/remarkjs/remark-gfm)
-- [remark-smartypants](https://github.com/silvenon/remark-smartypants)
-- [rehype-slug](https://github.com/rehypejs/rehype-slug)
-
-
-> ⚙️ You can include a custom Markdown parser inside `astro.config.mjs` by providing a function that follows the `MarkdownParser` type declared inside [this file](https://github.com/withastro/astro/blob/main/packages/astro/src/@types/astro.ts).
-
-```js
-// astro.config.mjs
-export default {
-  markdown: {
-    // ...
-  },
-};
-```
+You can customize the Markdown parsing by modifing your `astro.config.mjs`. [Here you can read the full reference](/en/reference/configuration-reference/#markdown-options).
 
 ### Remark and Rehype Plugins
 
-Astro supports third-party plugins for Markdown. You can provide your plugins in `astro.config.mjs`.
+Astro supports third-party [remark](https://github.com/remarkjs/remark) and [rehype](https://github.com/rehypejs/rehype) plugins for Markdown. You can provide your plugins in `astro.config.mjs`.
 
-> **Note:** Enabling custom `remarkPlugins` or `rehypePlugins` removes Astro’s built-in support for the plugins previously mentioned. You must explicitly add these plugins to your `astro.config.mjs` file, if desired.
+By default, Astro comes with [GitHub-flavored Markdown](https://github.com/remarkjs/remark-gfm) and [remark-smartypants](https://github.com/silvenon/remark-smartypants) pre-enabled. Take into account that enabling custom `remarkPlugins` or `rehypePlugins` will remove these built-in plugins and you would need to explicitly add these plugins if desired.
```

Comment:
The fact that this disables defaults seems important, so maybe it’s worth preserving the extra formatting to highlight it.

```suggestion
> **Note:** By default, Astro comes with [GitHub-flavored Markdown](https://github.com/remarkjs/remark-gfm) and [remark-smartypants](https://github.com/silvenon/remark-smartypants) pre-enabled. Enabling custom `remarkPlugins` or `rehypePlugins` will remove these built-in plugins and you need to explicitly add these plugins if desired.
```